### PR TITLE
Add and translate new strings to PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -87,7 +87,8 @@
     "waitingForDeposit": "Aguardando depósito",
     "exchanging": "Nós estamos trocando seu BTC para rBTC.",
     "transfered": "foi transferido para sua carteira RSK",
-    "swapAgain": "Realizar nova troca"
+    "swapAgain": "Realizar nova troca",
+    "disabledText": "Top-up deposits have been temporarily disabled. We ask that you not send funds until re-enabled. Thank you!"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:91]:
```suggestion
    "disabledText": "Os depósitos foram temporariamente desativados. Pedimos que você não envie fundos até que seja reativado. Obrigado!"
```

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -311,6 +312,8 @@
   "rskConnectTutorial": {
     "browser_wallet": "Browser Wallet",
     "mobile_wallet": "Mobile Wallet",
+    "mobile_wallet_alert_line1": "Make sure to use mobile wallet that supports RSK wallet. We suggest RWallet or Defiant.",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:315]:
```suggestion
    "mobile_wallet_alert_line1": "Certifique-se de usar uma carteira compatível com a rede RSK. Sugerimos a RWallet ou a Defiant.",
```

File: src/locales/pt_br/translation.json [hunk 3]
```diff
@@ -311,6 +312,8 @@
   "rskConnectTutorial": {
     "browser_wallet": "Browser Wallet",
     "mobile_wallet": "Mobile Wallet",
+    "mobile_wallet_alert_line1": "Certifique-se de usar uma carteira compatível com a rede RSK. Sugerimos a RWallet ou a Defiant.",
+    "mobile_wallet_alert_line2": "MetaMask has issues with calculating gas price for RSK network - always review if it's not to large before approving transaction!",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:316]:
```suggestion
    "mobile_wallet_alert_line2": "A MetaMask tem problemas com o cálculo das taxas de transação quando usada com a rede RSK - sempre revise se não está pagando muito antes de aprovar a transação!",
```

File: src/locales/pt_br/translation.json [hunk 4]
```diff
@@ -360,6 +363,63 @@
       "4": {
         "title": "Connect With Mobile Wallet",
         "banner": "For any assistance, reach out to the community"
+      },
+      "5": {
+        "title": "Choose Your Wallet",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:368]:
```suggestion
        "title": "Escolha sua carteira",
```

File: src/locales/pt_br/translation.json [hunk 5]
```diff
@@ -360,6 +363,63 @@
       "4": {
         "title": "Connect With Mobile Wallet",
         "banner": "For any assistance, reach out to the community"
+      },
+      "5": {
+        "title": "Escolha sua carteira",
+        "banner": "For any assistance, reach out to the community"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:369]:
```suggestion
        "banner": "Para obter ajuda, entre em contato com a comunidade"
```

File: src/locales/pt_br/translation.json [hunk 6]
```diff
@@ -360,6 +363,63 @@
...
         "banner": "For any assistance, reach out to the community"
+      },
+      "5": {
+        "title": "Escolha sua carteira",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      }
+    }
+  },
+   "SOVConnectTutorial": {
+    "browser_wallet": "Browser Wallet",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:374]:
```suggestion
    "browser_wallet": "Carteira de navegador",
```

File: src/locales/pt_br/translation.json [hunk 7]
```diff
@@ -360,6 +363,63 @@
...
+      },
+      "5": {
+        "title": "Escolha sua carteira",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      }
+    }
+  },
+   "SOVConnectTutorial": {
+    "browser_wallet": "Carteira de navegador",
+    "mobile_wallet": "Mobile Wallet",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:375]:
```suggestion
    "mobile_wallet": "Carteira de celular",
```

File: src/locales/pt_br/translation.json [hunk 8]
```diff
@@ -360,6 +363,63 @@
...
+      "5": {
+        "title": "Escolha sua carteira",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      }
+    }
+  },
+   "SOVConnectTutorial": {
+    "browser_wallet": "Carteira de navegador",
+    "mobile_wallet": "Carteira de celular",
+    "hardware_wallet": "Hardware Wallet",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:376]:
```suggestion
    "hardware_wallet": "Carteira física",
```

File: src/locales/pt_br/translation.json [hunk 9]
```diff
@@ -360,6 +363,63 @@
...
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      }
+    }
+  },
+   "SOVConnectTutorial": {
+    "browser_wallet": "Carteira de navegador",
+    "mobile_wallet": "Carteira de celular",
+    "hardware_wallet": "Carteira física",
+    "speech": {
+      "1": "Open up your browser wallet",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:378]:
```suggestion
      "1": "Abra sua carteira de navegador",
```

File: src/locales/pt_br/translation.json [hunk 10]
```diff
@@ -360,6 +363,63 @@
...
+      }
+    }
+  },
+   "SOVConnectTutorial": {
+    "browser_wallet": "Carteira de navegador",
+    "mobile_wallet": "Carteira de celular",
+    "hardware_wallet": "Carteira física",
+    "speech": {
+      "1": "Abra sua carteira de navegador",
+      "2": "Go to network dropdown",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:379]:
```suggestion
      "2": "Vá para o menu rede",
```

File: src/locales/pt_br/translation.json [hunk 11]
```diff
@@ -360,6 +363,63 @@
...
+    }
+  },
+   "SOVConnectTutorial": {
+    "browser_wallet": "Carteira de navegador",
+    "mobile_wallet": "Carteira de celular",
+    "hardware_wallet": "Carteira física",
+    "speech": {
+      "1": "Abra sua carteira de navegador",
+      "2": "Vá para o menu rede",
+      "3": "Select custom RPC",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:380]:
```suggestion
      "3": "Selecione RPC personalizado",
```

File: src/locales/pt_br/translation.json [hunk 12]
```diff
@@ -360,6 +363,63 @@
...
+      "4": "Input network details",
+      "5": "Click Save",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:382]:
```suggestion
      "5": "Clique em salvar",
```

File: src/locales/pt_br/translation.json [hunk 13]
```diff
@@ -360,6 +363,63 @@
...
+    "browser_wallet": "Carteira de navegador",
+    "mobile_wallet": "Carteira de celular",
+    "hardware_wallet": "Carteira física",
+    "speech": {
+      "1": "Abra sua carteira de navegador",
+      "2": "Vá para o menu rede",
+      "3": "Selecione RPC personalizado",
+      "4": "Input network details",
+      "5": "Clique em salvar",
+      "6": "Click Add Network",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:383]:
```suggestion
      "6": "Clique em adicionar rede",
```

File: src/locales/pt_br/translation.json [hunk 14]
```diff
@@ -360,6 +363,63 @@
...
+    "mobile_wallet": "Carteira de celular",
+    "hardware_wallet": "Carteira física",
+    "speech": {
+      "1": "Abra sua carteira de navegador",
+      "2": "Vá para o menu rede",
+      "3": "Selecione RPC personalizado",
+      "4": "Input network details",
+      "5": "Clique em salvar",
+      "6": "Clique em adicionar rede",
+      "7": "Engage Your Wallet!"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:384]:
```suggestion
      "7": "Conecte sua carteira!"
```

File: src/locales/pt_br/translation.json [hunk 15]
```diff
@@ -360,6 +363,63 @@
...
+    },
+    "mobileSpeech": {
+      "1": "Go to Wallet Menu",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:387]:
```suggestion
      "1": "Vá para o menu da carteira",
```

File: src/locales/pt_br/translation.json [hunk 16]
```diff
@@ -360,6 +363,63 @@
...
+      "2": "Vá para o menu rede",
+      "3": "Selecione RPC personalizado",
+      "4": "Input network details",
+      "5": "Clique em salvar",
+      "6": "Clique em adicionar rede",
+      "7": "Conecte sua carteira!"
+    },
+    "mobileSpeech": {
+      "1": "Vá para o menu da carteira",
+      "2": "Select Settings",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:388]:
```suggestion
      "2": "Selecione configurações",
```

File: src/locales/pt_br/translation.json [hunk 17]
```diff
@@ -360,6 +363,63 @@
...
+      "3": "Select Networks",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:389]:
```suggestion
      "3": "Selecione redes",
```

File: src/locales/pt_br/translation.json [hunk 18]
```diff
@@ -360,6 +363,63 @@
...
+      "4": "Input network details",
+      "5": "Clique em salvar",
+      "6": "Clique em adicionar rede",
+      "7": "Conecte sua carteira!"
+    },
+    "mobileSpeech": {
+      "1": "Vá para o menu da carteira",
+      "2": "Selecione configurações",
+      "3": "Select Networks",
+      "4": "Click Add Network",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:390]:
```suggestion
      "4": "Clique em adicionar rede",
```

File: src/locales/pt_br/translation.json [hunk 19]
```diff
@@ -360,6 +363,63 @@
...
+      "6": "Clique em adicionar rede",
+      "7": "Conecte sua carteira!"
+    },
+    "mobileSpeech": {
+      "1": "Vá para o menu da carteira",
+      "2": "Selecione configurações",
+      "3": "Selecione redes",
+      "4": "Clique em adicionar rede",
+      "5": "Input Network Settings",
+      "6": "Click Add",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:392]:
```suggestion
      "6": "Clique em adicionar",
```

File: src/locales/pt_br/translation.json [hunk 20]
```diff
@@ -360,6 +363,63 @@
...
+      "7": "Scan QR Code to Connect!"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:393]:
```suggestion
      "7": "Ler código QR para conectar!"
```

File: src/locales/pt_br/translation.json [hunk 21]
```diff
@@ -360,6 +363,63 @@
...
+    "mobileSpeech": {
+      "1": "Vá para o menu da carteira",
+      "2": "Selecione configurações",
+      "3": "Selecione redes",
+      "4": "Clique em adicionar rede",
+      "5": "Input Network Settings",
+      "6": "Clique em adicionar",
+      "7": "Ler código QR para conectar!"
+    },
+    "speech_qr_code": "Scan QR code with a wallet-connect compatible wallet",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:395]:
```suggestion
    "speech_qr_code": "Ler código QR com uma carteira compatível com a função wallet-connect",
```

File: src/locales/pt_br/translation.json [hunk 22]
```diff
@@ -360,6 +363,63 @@
...
+      "2": "Selecione configurações",
+      "3": "Selecione redes",
+      "4": "Clique em adicionar rede",
+      "5": "Input Network Settings",
+      "6": "Clique em adicionar",
+      "7": "Ler código QR para conectar!"
+    },
+    "speech_qr_code": "Ler código QR com uma carteira compatível com a função wallet-connect",
+    "input_settings": {
+      "title": "SOV TOKEN SETTINGS",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:397]:
```suggestion
      "title": "CONFIGURAÇÕES DO ATIVO SOV",
```

File: src/locales/pt_br/translation.json [hunk 23]
```diff
@@ -360,6 +363,63 @@
...
+      "token_address": "Token Contract Address",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:398]:
```suggestion
      "token_address": "Endereço do contrato do ativo",
```

File: src/locales/pt_br/translation.json [hunk 24]
```diff
@@ -360,6 +363,63 @@
...
+      "4": "Clique em adicionar rede",
+      "5": "Input Network Settings",
+      "6": "Clique em adicionar",
+      "7": "Ler código QR para conectar!"
+    },
+    "speech_qr_code": "Ler código QR com uma carteira compatível com a função wallet-connect",
+    "input_settings": {
+      "title": "CONFIGURAÇÕES DO ATIVO SOV",
+      "token_address": "Token Contract Address",
+      "symbol": "Symbol",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:399]:
```suggestion
      "symbol": "Símbolo",
```

File: src/locales/pt_br/translation.json [hunk 25]
```diff
@@ -360,6 +363,63 @@
...
+      "5": "Input Network Settings",
+      "6": "Clique em adicionar",
+      "7": "Ler código QR para conectar!"
+    },
+    "speech_qr_code": "Ler código QR com uma carteira compatível com a função wallet-connect",
+    "input_settings": {
+      "title": "CONFIGURAÇÕES DO ATIVO SOV",
+      "token_address": "Endereço do contrato do ativo",
+      "symbol": "Símbolo",
+      "decimals": "Decimals"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:400]:
```suggestion
      "decimals": "Decimais"
```

File: src/locales/pt_br/translation.json [hunk 26]
```diff
@@ -360,6 +363,63 @@
...
+    },
+    "step": "Step",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:402]:
```suggestion
    "step": "Passo",
```

File: src/locales/pt_br/translation.json [hunk 27]
```diff
@@ -360,6 +363,63 @@
...
+    "input_settings": {
+      "title": "CONFIGURAÇÕES DO ATIVO SOV",
+      "token_address": "Endereço do contrato do ativo",
+      "symbol": "Símbolo",
+      "decimals": "Decimais"
+    },
+    "step": "Step",
+    "screens": {
+      "1": {
+        "title": "Connect Your Wallet",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:405]:
```suggestion
        "title": "Conecte sua carteira",
```

File: src/locales/pt_br/translation.json [hunk 28]
```diff
@@ -360,6 +363,63 @@
...
+      "title": "CONFIGURAÇÕES DO ATIVO SOV",
+      "token_address": "Endereço do contrato do ativo",
+      "symbol": "Símbolo",
+      "decimals": "Decimais"
+    },
+    "step": "Passo",
+    "screens": {
+      "1": {
+        "title": "Conecte sua carteira",
+        "banner": "For any assistance, reach out to the community"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:406]:
```suggestion
        "banner": "Para obter ajuda, entre em contato com a comunidade"
```

File: src/locales/pt_br/translation.json [hunk 29]
```diff
@@ -360,6 +363,63 @@
...
+      },
+      "11": {
+        "title": "Connect to the RSK Network",
+        "banner": "For any assistance, reach out to the community"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:410]:
```suggestion
        "banner": "Para obter ajuda, entre em contato com a comunidade"
```

File: src/locales/pt_br/translation.json [hunk 30]
```diff
@@ -360,6 +363,63 @@
...
+      },
+      "2": {
+        "title": "How to Connect to RSK Network",
+        "banner": "For any assistance, reach out to the community"
+      },
+      "3": {
+        "title": "Scan QR Code",
+        "banner": "For any assistance, reach out to the community"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:418]:
```suggestion
        "banner": "Para obter ajuda, entre em contato com a comunidade"
```

File: src/locales/pt_br/translation.json [hunk 31]
```diff
@@ -360,6 +363,63 @@
...
+      "decimals": "Decimais"
+    },
+    "step": "Passo",
+    "screens": {
+      "1": {
+        "title": "Conecte sua carteira",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      },
+      "11": {
+        "title": "Connect to the RSK Network",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:409]:
```suggestion
        "title": "Conecte na rede RSK",
```

File: src/locales/pt_br/translation.json [hunk 32]
```diff
@@ -360,6 +363,63 @@
...
+      "1": {
+        "title": "Conecte sua carteira",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      },
+      "11": {
+        "title": "Conecte na rede RSK",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      },
+      "2": {
+        "title": "How to Connect to RSK Network",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:413]:
```suggestion
        "title": "Como se conectar à rede RSK",
```

File: src/locales/pt_br/translation.json [hunk 33]
```diff
@@ -360,6 +363,63 @@
...
+  },
+   "SOVConnectTutorial": {
+    "browser_wallet": "Carteira de navegador",
+    "mobile_wallet": "Carteira de celular",
+    "hardware_wallet": "Carteira física",
+    "speech": {
+      "1": "Abra sua carteira de navegador",
+      "2": "Vá para o menu rede",
+      "3": "Selecione RPC personalizado",
+      "4": "Input network details",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:381]:
```suggestion
      "4": "Insira os detalhes da rede",
```

File: src/locales/pt_br/translation.json [hunk 34]
```diff
@@ -360,6 +363,63 @@
...
+      "5": "Clique em salvar",
+      "6": "Clique em adicionar rede",
+      "7": "Conecte sua carteira!"
+    },
+    "mobileSpeech": {
+      "1": "Vá para o menu da carteira",
+      "2": "Selecione configurações",
+      "3": "Selecione redes",
+      "4": "Clique em adicionar rede",
+      "5": "Input Network Settings",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:391]:
```suggestion
      "5": "Insira as configurações da rede",
```

File: src/locales/pt_br/translation.json [hunk 35]
```diff
@@ -360,6 +363,63 @@
...
+      "11": {
+        "title": "Conecte na rede RSK",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      },
+      "2": {
+        "title": "Como se conectar à rede RSK",
+        "banner": "For any assistance, reach out to the community"
+      },
+      "3": {
+        "title": "Scan QR Code",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:417]:
```suggestion
        "title": "Leia o código QR",
```

File: src/locales/pt_br/translation.json [hunk 36]
```diff
@@ -360,6 +363,63 @@
...
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      },
+      "4": {
+        "title": "How to connect SOV to your Metamask wallet",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:421]:
```suggestion
        "title": "Como conectar SOV à sua carteira Metamask",
```

File: src/locales/pt_br/translation.json [hunk 37]
```diff
@@ -360,6 +363,63 @@
...
+        "title": "Conecte sua carteira",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      },
+      "11": {
+        "title": "Conecte na rede RSK",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      },
+      "2": {
+        "title": "Como se conectar à rede RSK",
+        "banner": "For any assistance, reach out to the community"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:414]:
```suggestion
        "banner": "Para obter ajuda, entre em contato com a comunidade"
```

File: src/locales/pt_br/translation.json [hunk 38]
```diff
@@ -452,6 +512,12 @@
       "description": "Informações sobre a carteira conectada"
     }
   },
+  "salesPage": {
+    "meta": {
+      "title": "Presale",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:517]:
```suggestion
      "title": "Pré-venda",
```

File: src/locales/pt_br/translation.json [hunk 39]
```diff
@@ -452,6 +512,12 @@
...
+      "description": "SOV Presale Page"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:518]:
```suggestion
      "description": "Página de pré-venda SOV"
```

File: src/locales/pt_br/translation.json [hunk 40]
```diff
@@ -360,6 +363,63 @@
...
+        "title": "Como se conectar à rede RSK",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      },
+      "3": {
+        "title": "Leia o código QR",
+        "banner": "Para obter ajuda, entre em contato com a comunidade"
+      },
+      "4": {
+        "title": "Como conectar SOV à sua carteira Metamask",
+        "banner": "For any assistance, reach out to the community"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:422]:
```suggestion
        "banner": "Para obter ajuda, entre em contato com a comunidade"
```

**Comment by bitcoinuser** [comment]
Hi @creed-victor please merge this one. Thanks.

File: src/locales/pt_br/translation.json [hunk 41]
```diff
@@ -309,68 +310,127 @@
...
+      "7": "Conecte sua carteira!"
+    },
+    "mobileSpeech": {
+      "1": "Vá para o menu da carteira",
+      "2": "Selecione configurações",
+      "3": "Selecione redes",
+      "4": "Clique em adicionar rede",
+      "5": "Insira as configurações da rede",
+      "6": "Clique em adicionar",
+      "7": "Ler código QR para conectar!"
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:393]:
```suggestion
      "7": "Leia o código QR para conectar!"
```

File: src/locales/pt_br/translation.json [hunk 42]
```diff
@@ -87,7 +87,8 @@
     "waitingForDeposit": "Aguardando depósito",
     "exchanging": "Nós estamos trocando seu BTC para rBTC.",
     "transfered": "foi transferido para sua carteira RSK",
-    "swapAgain": "Realizar nova troca"
+    "swapAgain": "Realizar nova troca",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:90]:
```suggestion
    "swapAgain": "Realizar outra troca",
```

**Review by creed-victor — APPROVED** [review]

**Pull request merged**